controllers/AppController.js

```diff
@@ -0,0 +1,13 @@
+import reverseString from '../utils/reverseString.js';
+
+class AppController {
+  async app(req, res) {
```

Comment:
These methods are marked as `async` but there is no `await`

Reply:
Yeah, these are useless here. Will remove them